js/languagePickerDrawerView.js

```diff
@@ -67,6 +66,12 @@ define([
       Adapt.trigger('drawer:closeDrawer');
     },
 
+    onPopupOpened: function () {
+      // move popup close focus to #focuser
+      Adapt.a11y._popup._focusStack.pop();
+      Adapt.a11y._popup._focusStack.push($('#a11y-focuser'));
```

Comment:
```suggestion
      Adapt.a11y.setPopupCloseTo($('#a11y-focuser'));
```
https://github.com/adaptlearning/adapt_framework/blob/c0e308f5302056bc8a48dfd055d6f1669a7ba944/src/core/js/a11y.js#L721

https://github.com/adaptlearning/adapt_framework/blob/c0e308f5302056bc8a48dfd055d6f1669a7ba944/src/core/js/a11y/popup.js#L213-L215